README.md

```diff
@@ -1,2 +1,2 @@
 # battleship-etv
-A mock project to get familiar with the development and collaboration process. 
+# A mock project to get familiar with the development and collaboration process. 
```

Comment:
Notice you accidentally changed the README.md file and it was added to your changes. Though this is a harmless change be weary of this next time and review what changes are being commited before you commit them.